scripts/add_to_whitelist.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+
+def main():
+    # Get words from command-line arguments
+    new_words = sys.argv[1:]
+
+    if not new_words:
+        print("Usage: ./whitelist_manager.py <word1> <word2> <word3> ...")
+        print("Example: ./whitelist_manager.py foo bar baz")
+        sys.exit(1)
+
+    whitelist_file = 'whitelist.txt'
```

Comment:
This will be relative to the current working directory, so something like the following would give unexpected results:

```bash
$ cd scripts
$ ./add_to_whitelist foobar
```